### ISSUE TITLE
Invalid pyproject.tom when migrating from Poetry

### ISSUE THREAD

**Issue by frafra** [pdm-project/pdm#723]
- [X] I have searched the issue tracker and believe that this is not a duplicate.

## Steps to reproduce

Run `pdm init` on a Poetry project and choose to reuse the existing `pyproject.toml`.

## Actual behavior

```
+[project]
+name = ""
+version = ""
+description = ""
+authors = [
+    {name = "Francesco Frassinelli", email = "francesco.frassinelli@nina.no"},
+    name = "Francesco Frassinelli"
+email = "francesco.frassinelli@nina.no"
+
+]
```

## Expected behavior

`authors` should contain a valid object.
```
Traceback (most recent call last):
  File "/home/francesco.frassinell/.local/bin/pdm", line 8, in <module>
    sys.exit(main())
  File "/home/francesco.frassinell/.local/pipx/venvs/pdm/lib/python3.10/site-packages/pdm/core.py", line 200, in main
    return Core().main(args)
  File "/home/francesco.frassinell/.local/pipx/venvs/pdm/lib/python3.10/site-packages/pdm/core.py", line 138, in main
    self.ensure_project(options, obj)
  File "/home/francesco.frassinell/.local/pipx/venvs/pdm/lib/python3.10/site-packages/pdm/core.py", line 108, in ensure_project
    migrate_pyproject(options.project)
  File "/home/francesco.frassinell/.local/pipx/venvs/pdm/lib/python3.10/site-packages/pdm/cli/actions.py", line 684, in migrate_pyproject
    if project.pyproject and "project" in project.pyproject:
  File "/home/francesco.frassinell/.local/pipx/venvs/pdm/lib/python3.10/site-packages/pdm/project/core.py", line 99, in pyproject
    data = atoml.parse(self.pyproject_file.read_text("utf-8"))
  File "/home/francesco.frassinell/.local/pipx/venvs/pdm/lib/python3.10/site-packages/atoml/api.py", line 68, in parse
    return Parser(string).parse()
  File "/home/francesco.frassinell/.local/pipx/venvs/pdm/lib/python3.10/site-packages/atoml/parser.py", line 152, in parse
    key, value = self._parse_table()
  File "/home/francesco.frassinell/.local/pipx/venvs/pdm/lib/python3.10/site-packages/atoml/parser.py", line 1140, in _parse_table
    item = self._parse_item()
  File "/home/francesco.frassinell/.local/pipx/venvs/pdm/lib/python3.10/site-packages/atoml/parser.py", line 290, in _parse_item
    return self._parse_key_value(True)
  File "/home/francesco.frassinell/.local/pipx/venvs/pdm/lib/python3.10/site-packages/atoml/parser.py", line 383, in _parse_key_value
    val = self._parse_value()
  File "/home/francesco.frassinell/.local/pipx/venvs/pdm/lib/python3.10/site-packages/atoml/parser.py", line 555, in _parse_value
    return self._parse_array()
  File "/home/francesco.frassinell/.local/pipx/venvs/pdm/lib/python3.10/site-packages/atoml/parser.py", line 723, in _parse_array
    raise self.parse_error(UnexpectedCharError, self._current)
atoml.exceptions.UnexpectedCharError: Unexpected character: 'n' at line 35 col 4
```

`name, `version` and `description` are not copied over.

## Environment Information

```bash
pdm info && pdm info --env
PDM version:        1.10.0
Python Interpreter: /home/francesco.frassinell/Software/bin/python3 (3.10)
Project Root:       /home/francesco.frassinell/Code/coat-tools
Project Packages:   /home/francesco.frassinell/Code/coat-tools/__pypackages__/3.10
{
  "implementation_name": "cpython",
  "implementation_version": "3.10.0",
  "os_name": "posix",
  "platform_machine": "x86_64",
  "platform_release": "4.15.0-159-generic",
  "platform_system": "Linux",
  "platform_version": "#167-Ubuntu SMP Tue Sep 21 08:55:05 UTC 2021",
  "python_full_version": "3.10.0",
  "platform_python_implementation": "CPython",
  "python_version": "3.10",
  "sys_platform": "linux"
}
```

**Comment by frostming** [comment]
What is the version of `atoml`?
Run: `/home/francesco.frassinell/.local/pipx/venvs/pdm/bin/pip list` to have a look

**Comment by frafra** [comment]
`pip` is not there, but I can see that `atoml` version is `1.1.0`:
```
$ echo /home/francesco.frassinell/.local/pipx/venvs/pdm/lib/python3.10/site-packages/atoml*
/home/francesco.frassinell/.local/pipx/venvs/pdm/lib/python3.10/site-packages/atoml /home/francesco.frassinell/.local/pipx/venvs/pdm/lib/python3.10/site-packages/atoml-1.1.0.dist-info
```

**Comment by frostming** [comment]
Can you provide the original input?

**Comment by frostming** [comment]
Nevermind, it was fixed in d813be6
